tests/utils.py

```diff
@@ -21,7 +21,7 @@ def item_exists(item_pid):
     return True
 
 
-def is_item_available(item_pid):
+def item_can_circulate(item_pid):
     """Return if item is available for checkout."""
```

Comment:
I would just check the comment to be aligned with the meaning of this method, something like:
`Return True if the item can circulate, False otherwise.`